### PR TITLE
Screenshots & some small changes

### PR DESCRIPTION
**Pull request by petrmanak**

### CODE REVIEW

File: libs/langchain/langchain/tools/playwright/click_by_text.py [hunk 1]
```diff
@@ -102,15 +103,12 @@ async def _arun(
                 with open('tempfile', 'a') as f:
```
Comment (jakub-profiq) [inline libs/langchain/langchain/tools/playwright/click_by_text.py]: We have more writing to the file :)

Comment (petrmanak) [reply libs/langchain/langchain/tools/playwright/click_by_text.py]: Damn...

**Review by jakub-profiq — APPROVED** [review]
Good job. Lets merge it! :)

File: libs/langchain/langchain/tools/playwright/expect_text.py [hunk 2]
```diff
@@ -57,16 +56,13 @@ async def _arun(
         if self.async_browser is None:
             raise ValueError(f"Asynchronous browser not provided to {self.name}")
         page = await aget_current_page(self.async_browser)
+        playwright_cmd = "await expect(page.getByText(/{text}/).nth({index})).toHaveText(/{text}/);\n"
```
Comment (jakub-profiq) [inline libs/langchain/langchain/tools/playwright/expect_text.py]: Add f"..."

File: libs/langchain/langchain/tools/playwright/click_by_text.py (hunk 1 above)
Comment (petrmanak) [reply libs/langchain/langchain/tools/playwright/click_by_text.py]: fixed

File: libs/langchain/langchain/tools/playwright/expect_text.py (hunk 2 above)
Comment (petrmanak) [reply libs/langchain/langchain/tools/playwright/expect_text.py]: fixed

**Pull request merged**